.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,7 @@ Brief description of what this PR does, and why it is needed.
 - [ ] Symlinks from new migrations present or corrected for any new migrations
 - [ ] PR has a name that won't get you publicly shamed for vagueness
 - [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`
+- [ ] If not merging to an anchor PR, any new DAO functions have tests
```

Comment:
I think this is maybe a little broad, and a little narrow.

I think it's too broad in the sense that we want tests for two specific things -- complex behaviors that we want to document intent for (like when to kick off ingests with new scenes) and new sql strings that we can't prove are correct without exercising them. If you have a new `Dao` method that just combines other `Dao` methods, I don't think it necessarily needs tests.

But I also think it's too narrow, in that I think PRs against anchor branches should still have tests. Why should they be exempted? Maybe it's time to revisit our [testing philosophy](https://github.com/raster-foundry/raster-foundry/blob/develop/docs/architecture/adr-0003-testing-philosophy.md)

Reply:
I agree with @jisantuc on this one. Just to emphasize I think that any new SQL string of any size (even an interpolated filter) should be tested.

Reply:
Yeah I think that's pretty good as a solution here

Reply:
We should also think about what should go into that testing philosophy doc -- before the question was something like "how should we test all the things we're going to test," but I think now it's more along the lines of "what parts of the universe should we test". Like, not having API tests is mostly fine, since we test so much of the database. Not testing querying with filters probably isn't (see, e.g., https://github.com/raster-foundry/raster-foundry/issues/3356).

Reply:
:+1: